guides/gatherers.md

```diff
@@ -119,16 +158,17 @@ For extra information refer to [trento-project/agent/../gatherers/corosyncconf_t
 
 This gatherer allows accessing the output of the `corosync-cmapctl` tool. It supports all of the keys returned by it to be queried.
 
+Example arguments:
 
-Sample arguments:
+| Name                         | Return value                     |
+| :--------------------------- | :------------------------------- |
+| `totem.token`                | extracted value from the command |
```

Comment:
Now that we have the new implementation of this gatherer done, we should include new examples. @jamie-suse could you do that in an additional PR?

Reply:
Yep @arbulu89, I can add examples featuring an output that contains a list.